### ISSUE TITLE
Status of lock - In my country have different side of locker, when the door open the status show locked

### ISSUE THREAD

**Issue by antonbek89** [rospogrigio/airbnk_mqtt#23]
How do I need to solve this situation?

**Comment by Lexa2414118** [comment]
I have the same trouble

**Comment by dkwireless** [comment]
When you change direction in app it does not change in integration. 
So in app is now locked but in integration shows open. I think direction is also reversed, but that is not an issue.
Is there a way to add toggle or some setting to integration, if it's not too big trouble.
Anyway, I guess we could use some template entity and reverse output.

**Comment by rospogrigio** [comment]
Yeah I know of this request but I never had the time to code it, sorry. Too busy in this period.
